data/styles/clear/include/Basemap.mapcss

```diff
@@ -136,19 +136,17 @@ area|z14-[landuse=village_green],
 area|z14-[landuse=field],
 {fill-color: @green1;}
 area|z11-12[landuse=forest],
-area|z12[leisure=garden],
-{fill-color: @green2;}
+area|z12-[leisure=garden],
+{fill-color: @green5;}
```

Comment:
its good that gardens will have colors distinct from the forests now!

but before z15 garden will be darker than forests which is not logical IMO

and on z15 it'll be the same green5 for both

and from z16- a forest will be darker than a garden (which is fine IMO)